---
layout: default
title: The Van Doorslaer lab at the University of Arizona
---

<body>
<div id="wrap">
  <div id="masthead">
    <h1>{{page.title}}</h1>
  </div>
  <div id="menucontainer">
    {% include nav.html %}
  </div>
	{% include sidebar.html %}
    <div id="content">
    
<h3>Current projects in the lab</h3>
<div class="page-break">

<img src="/images/projects/scRNA.png" alt="stack" width="314.6" height="121" align="left" class="lazy left">
<p align="justify">
	<H4>Host factors regulating persistence. </H4> Until recently, it has been challenging to study the HPV lifecycle following infection. We and others have optimized infection of patient-derived primary cells to investigate how host-virus interactions contribute to initial genome amplification during the establishment phase and long-term viral persistence. Whether a virus persists could (1) be purely stochastic, (2) depend on some pre-existing cellular state (e.g., baseline activation of innate immunity), (3) rely on differential viral gene expression, or (4) (likely) a combination of these. To investigate this heterogeneity in outcomes, we used single-cell RNA Seq to interrogate the transcriptional profile of 5000 infected primary human cervical-derived keratinocytes with HPV18. We identified hundreds of human genes differentially regulated in HPV18 infected cells. To my knowledge, we are the only lab in the world using single-cell RNA-Seq to study how the virus manipulates individual cells. The paper describing these data is currently in preparation. 
We are currently following up on one of the identified targets. Since an inhibitor for this pathway is currently in clinical trials for non-HPV tumors, this project has immediate translational potential. 
<b>This research is the focus of a funded Arizona Biomedical Research Centre (ABRC) grant, a new NIAID R01 grant, and a pending NIAID R21 proposal </b>. This project also illustrates our lab philosophy to translate large scale genomics data into actionable biochemistry and cell biology research.
</p>

<br><br>
<img src="/images/projects/TLR9.png" alt="stack" width="227.6" height="200" align="right" class = "lazy right">
<p align="justify">
<H4> Role of the innate immune system during persistent infection. </H4> Cellular innate immune pathways exist to restrict viral infections. Innate immune responses are initiated after recognizing molecular products associated with microbial infections, known as pathogen-associated molecular patterns (PAMPs). More recent data have illustrated that in addition to PAMPs, DNA generated from cellular damage (host damage-associated molecular patterns (DAMPs)) activate the innate immune response. 
In collaboration with the lab of <a target="_blank" href="https://cancercenter.arizona.edu/person/samuel-k-campos-phd"<b>Dr. Campos</b></a>, my lab studies how HPV manipulates the <b>cGAS/STING</b> arm of the innate immune response. Cytosolic DNA activates the cGAS/STING pathway. This initiates a signaling cascade that results in an antiviral response, detrimental to the establishment and persistence of HPV infections. The virus must have developed mechanisms to counteract cellular cGAS/STING responses. Dr. Campos and I have published a (<a target="_blank" href="https://journals.plos.org/plospathogens/article/authors?id=10.1371/journal.ppat.1009028"<b>paper (Uhlorn et al., 2020)</b></a> on this pathway (Co-corresponding authors) and are preparing another manuscript to present this data. This work is supported by a new <b>American Cancer Society</b> grant to my lab. Dr. Campos and I have received funding to support this project in our labs. We also have a pending R01 grant studying these interactions.
We are also interested in how papillomaviruses have evolved to evade detection by <b>TLR9 and APOBEC3</b>. In recently published papers (<a target="_blank" href="https://academic.oup.com/ve/article/1/1/vev015/2568703"<b>Warren et al., 2015</b></a> and <a target="_blank" href="https://journals.asm.org/doi/full/10.1128/mbio.00054-22"<b>King et al., 2022</b></a>), we demonstrate how interaction with these innate immune sensors is impacting the evolutionary history of papillomaviruses. Indeed, we believe that the host-virus interactions are critical in driving the evolution of papillomaviruses. Interestingly, this evolutionary pressure appears to be primarily acting at the level of the viral DNA without altering viral protein composition. 
</p>

<br><br>
<img src="/images/projects/raft.png" alt="stack" width="231" height="200" align="left" class="lazy left">
<p align="justify"><H4>Viral manipulation of cellular differentiation. </H4>The ability to persist in actively dividing cells is essential to the viral lifecycle and oncogenicity. While a delay of cellular differentiation is critical, completion of the viral lifecycle depends on the terminal differentiation of the cell. Indeed, a differentiation-dependent mechanism ensures that 'late' structural genes are not expressed until viral replication has occurred. During keratinocyte differentiation, Spatio-temporal changes in gene expression determine epidermis structure and function. HPVs have evolved to alter this balance while intricately linking their lifecycle to this physiological process. These processes are poorly understood on the host and the viral side. Our overall goal is to determine the critical host-virus interactions that regulate the late stages of a differentiation-dependent HPV lifecycle. </p>
<p>We use a physiologically relevant system, <i>HPV16 positive tonsillar epithelium equivalents grown in 3D organotypic raft culture </i>, to <b>characterize the cellular genes regulating Spatio-temporal control of late viral gene expression</b>. Specifically, we are characterizing the transcriptome of individual cells derived from these epidermises using a single-cell RNASeq approach to characterize the host genes whose expression is highly correlated with late viral gene expression. In parallel, we are performing a genome-wide CRISPR/Cas9 knockout screen in these 3D organotypic raft cultures. Since our screening approach targets differentiated cells, this genome-wide genetic dropout screen will determine which cellular genes are crucial for late viral gene transcription. This <b>NIDCR R03 funded</b> research will generate multiple testable hypotheses addressing the dramatic increase in HPV-positive head and neck cancers in <b>future R01 level proposals</b>.

</p>

<br><br>
<img src="/images/projects/discovery.png" alt="stack" width="314.6" height="121" align="right" class="lazy right">
<p align="justify"><H4>Virus Discovery and Evolution.</H4> As the COVID19 pandemic illustrates, it is crucial to understand how viruses evolve. This requires that we know what viruses circulate in humans and animals. The discovery of novel viruses has been a substantial effort by my laboratory in collaboration with the lab of <a target="_blank" href=" https://varsanilab.org/"<b>Dr. Varsani</b></a>. We sample a wide array of animal and human tissues and use Next-Generation sequencing to identify the viruses in these samples. New viruses are further characterized and used to study the evolutionary history of the ‘virome’ (the total collection of viruses). <b>We published 15 papers on this topic</b>. This research is supported by seed funding from <b>the BIO5 institute and a USDA NIFA award to my lab</b>.
</p>

<br><br>
<img src="/images/projects/PaVEBookmark.jpg" alt="stack" width="314.6" height="121" align="left" class="lazy left">
<p align="justify">
<H4>Supporting evolutionary genomics of papillomaviruses. </H4> My research program integrates the evolution of papillomaviruses into biochemical and cell biology methods. I believe that this approach is unique and has the potential to revolutionize papillomavirus research. To support these efforts by the broader community, I was instrumental in developing an online database of papillomavirus sequences, the papillomavirus episteme (<a target="_blank" href="https://pave.niaid.nih.gov/"<b>PaVE; https://pave.niaid.nih</b></a>). The PaVE database is described in 2 papers (Van doorslaer et al., <a target="_blank" href="https://academic.oup.com/nar/article/41/D1/D571/1075043"<b> 2013</b></a> and <a target="_blank" href="https://academic.oup.com/nar/article/45/D1/D499/2770655"<b> 2017</b></a>) and has been accessed by more than <b>30,000 investigators in 135 countries</b>. Building on the PaVE database’s philosophy of open science, we are <b>actively developing additional bioinformatic tools to analyze viral sequence data</b> that leverage existing software and are available to the community through our lab's <a target="_blank" href=”https://github.com/KVD-lab/puma"<b> GitHub page</b></a>. For example, the field of 'viral metagenomics' has dramatically expanded our understanding of viral diversity. We have developed the PV annotation tool (PuMA; <a target="_blank" href= "https://academic.oup.com/ve/article/6/2/veaa068/5897427"<b> Pace et al., 2020</b></a>) to provide researchers with a convenient and reproducible method to annotate novel PVs. My lab continues to develop the PaVE database and associated software. 
	
</p>


</div>  

