---
layout: default
title: The Van Doorslaer lab at the University of Arizona
---

<body>
<div id="wrap">
  <div id="masthead">
    <h1>{{page.title}}</h1>
  </div>
  <div id="menucontainer">
    {% include nav.html %}
  </div>
	{% include sidebar.html %}
    <div id="content">
    
<h3>Current projects in the lab</h3>
<div class="page-break">

<img src="/images/projects/tree2heatmap.jpg" alt="stack" width="314.6" height="121" align="left" class="lazy left">
<p align="justify">
	<H4>Host factors regulating persistence. </H4> Until recently, it has been challenging to study the HPV lifecycle following infection. We and others have optimized infection of patient-derived primary cells to investigate how host-virus interactions contribute to initial genome amplification during the establishment phase and long-term viral persistence. Whether a virus persists could (1) be purely stochastic, (2) depend on some pre-existing cellular state (e.g., baseline activation of innate immunity), (3) rely on differential viral gene expression, or (4) (likely) a combination of these. To investigate this heterogeneity in outcomes, we used single-cell RNA Seq to interrogate the transcriptional profile of 5000 infected primary human cervical-derived keratinocytes with HPV18. We identified hundreds of human genes differentially regulated in HPV18 infected cells. To my knowledge, we are the only lab in the world using single-cell RNA-Seq to study how the virus manipulates individual cells. The paper describing these data is currently in preparation. 
We are currently following up on one of the identified targets. Since an inhibitor for this pathway is currently in clinical trials for non-HPV tumors, this project has immediate translational potential. 
<b>This research is the focus of a funded Arizona Biomedical Research Centre (ABRC) grant, a new NIAID R01 grant, and a pending NIAID R21 proposal </b>. This project also illustrates our lab philosophy to translate large scale genomics data into actionable biochemistry and cell biology research.
</p>

<br><br>
<img src="/images/projects/recombination.jpg" alt="stack" width="227.6" height="200" align="right" class = "lazy right">
<p align="justify">
<H4> Role of the innate immune system during persistent infection. </H4> Cellular innate immune pathways exist to restrict viral infections. Innate immune responses are initiated after recognizing molecular products associated with microbial infections, known as pathogen-associated molecular patterns (PAMPs). More recent data have illustrated that in addition to PAMPs, DNA generated from cellular damage (host damage-associated molecular patterns (DAMPs)) activate the innate immune response. 
In collaboration with the lab of <a target="_blank" href="https://cancercenter.arizona.edu/person/samuel-k-campos-phd"<b>Dr. Campos</b></a>, my lab studies how HPV manipulates the <b>cGAS/STING</b> arm of the innate immune response. Cytosolic DNA activates the cGAS/STING pathway. This initiates a signaling cascade that results in an antiviral response, detrimental to the establishment and persistence of HPV infections. The virus must have developed mechanisms to counteract cellular cGAS/STING responses. We demonstrate that activation of the cGAS/STING pathway limits HPV18â€™s ability to establish in the host. Dr. Campos and I have published a paper on this pathway (Co-corresponding authors) and are preparing another manuscript to present this data. This work is supported by a new American Cancer Society grant to my lab. Dr. Campos and I have received funding to support this project in our labs. We also have a pending R01 grant studying these interactions.
We are also interested in how papillomaviruses have evolved to evade detection by <b>TLR9 and APOBEC3</b>. In recently published papers (<a target="_blank" href="https://academic.oup.com/ve/article/1/1/vev015/2568703"<b>Warren et al., 2015</b></a> and <a target="_blank" href="https://journals.asm.org/doi/full/10.1128/mbio.00054-22"<b>King et al., 2022</b></a>), we demonstrate how interaction with these innate immune sensors is impacting the evolutionary history of papillomaviruses. Indeed, we believe that the host-virus interactions are critical in driving the evolution of papillomaviruses. Interestingly, this evolutionary pressure appears to be primarily acting at the level of the viral DNA without altering viral protein composition. 

	
	
	</p>

<br><br>
<img src="/images/projects/virus.jpg" alt="stack" width="231" height="200" align="left" class="lazy left">
<p align="justify">Viruses are the most abundant biological entity in every ecosystem. From the ocean to the human skin, viruses are extremely plentiful. We believe that the community dynamics of viruses is essential for the health of an ecosystem. However, measuring this dynamics is complicated. Viruses cannot be cultured in the absence of a host. Furthermore, viruses do not share genes that are common to all families of viruses. <b>Viral Metagenomic</b> analyses allows us to study which viral communities are present in specific ecosystems. This project is a collaboration with Drs. Arvind Varsani (ASU) and Christopher Buck (NCI).</p>



<br><br>
<img src="/images/projects/PaVEBookmark.jpg" alt="stack" width="314.6" height="121" align="right" class="lazy right">
<p align="justify">The <a target="_blank" href="http://pave.niaid.nih.gov/home"<b>Papillomavirus Episteme (PaVE)</b></a> was established in the lab of Alison A. Mcbride at NIAID. PaVE provides highly organized and curated papillomavirus genomics information and tools to the scientific community. We develop tools that interact with or integrate into the PaVE website and databases</p>


</div>  

