---
layout: default
title: The Van Doorslaer lab at the University of Arizona
---

<body>
<div id="wrap">
  <div id="masthead">
    <h1>{{page.title}}</h1>
  </div>
  <div id="menucontainer">
    {% include nav.html %}
  </div>
	{% include sidebar.html %}
    <div id="content">
    
<h3>Current projects in the lab</h3>
<div class="page-break">

<img src="/images/projects/tree2heatmap.jpg" alt="stack" width="314.6" height="121" align="left" class="lazy left">
<p align="justify">
	<H3>Host factors regulating persistence. </H3> Until recently, it has been challenging to study the HPV lifecycle following infection. We and others have optimized infection of patient-derived primary cells to investigate how host-virus interactions contribute to initial genome amplification during the establishment phase and long-term viral persistence. Whether a virus persists could (1) be purely stochastic, (2) depend on some pre-existing cellular state (e.g., baseline activation of innate immunity), (3) rely on differential viral gene expression, or (4) (likely) a combination of these. To investigate this heterogeneity in outcomes, we used single-cell RNA Seq to interrogate the transcriptional profile of 5000 infected primary human cervical-derived keratinocytes with HPV18. We identified hundreds of human genes differentially regulated in HPV18 infected cells. To my knowledge, we are the only lab in the world using single-cell RNA-Seq to study how the virus manipulates individual cells. The paper describing these data is currently in preparation. 
We are currently following up on one of the identified targets. Since an inhibitor for this pathway is currently in clinical trials for non-HPV tumors, this project has immediate translational potential. 
<b>This research is the focus of a funded Arizona Biomedical Research Centre (ABRC) grant, a new NIAID R01 grant, and a pending NIAID R21 proposal </b>. This project also illustrates our lab philosophy to translate large scale genomics data into actionable biochemistry and cell biology research.
</p>

<br><br>
<img src="/images/projects/recombination.jpg" alt="stack" width="227.6" height="200" align="right" class = "lazy right">
<p align="justify">Over the span of millions of years, papillomaviruses have evolved to optimize most, if not all, base-pairs of their 8kb genomes. It is, therefore, likely that <b>inter-type recombination</b> will be associated with a loss of viral fitness compared to the parental types. In the absence of some changing environmental factor, this reduced fitness limits the chances of recombinants to become fixed in the population. Indeed, recombination between distinct papillomavirus types has not been an important determinant of the papillomavirus evolutionary history. Based on evolutionary analysis, the early genes and late genes have evolved as distinct cassettes. Successful, ancient recombination events appear to have combined the regions of the genome coding for the capsid proteins from one parent with the regulatory proteins of the second parent. For the oncogenic viruses, two of these regulatory proteins (E6 and E7) have been shown to be potent oncogenes. This project studies the effects of recombination on viral fitness.</p>


<br><br>
<img src="/images/projects/virus.jpg" alt="stack" width="231" height="200" align="left" class="lazy left">
<p align="justify">Viruses are the most abundant biological entity in every ecosystem. From the ocean to the human skin, viruses are extremely plentiful. We believe that the community dynamics of viruses is essential for the health of an ecosystem. However, measuring this dynamics is complicated. Viruses cannot be cultured in the absence of a host. Furthermore, viruses do not share genes that are common to all families of viruses. <b>Viral Metagenomic</b> analyses allows us to study which viral communities are present in specific ecosystems. This project is a collaboration with Drs. Arvind Varsani (ASU) and Christopher Buck (NCI).</p>



<br><br>
<img src="/images/projects/PaVEBookmark.jpg" alt="stack" width="314.6" height="121" align="right" class="lazy right">
<p align="justify">The <a target="_blank" href="http://pave.niaid.nih.gov/home"<b>Papillomavirus Episteme (PaVE)</b></a> was established in the lab of Alison A. Mcbride at NIAID. PaVE provides highly organized and curated papillomavirus genomics information and tools to the scientific community. We develop tools that interact with or integrate into the PaVE website and databases</p>


</div>  

